tests/dummy/app/templates/docs/index.md

```diff
@@ -0,0 +1,16 @@
+# Installation
+
+```bash
+ember install ember-caluma-form-builder
+```
+
+# Usage
+
+`ember-caluma-form-builder` is a [routable engine](http://ember-engines.com).
+To include it in your application just mount it in your `app/router.js` and
```

Comment:
Maybe also include a snippet for the router?

Reply:
Uhm there are exactly two snippets :wink:

Reply:
Duh, nvmd